python/ray/tests/test_channel.py

```diff
@@ -32,6 +39,50 @@ def test_put_local_get(ray_start_regular):
         chan.end_read()
 
 
+@pytest.mark.skipif(
+    sys.platform != "linux" and sys.platform != "darwin",
+    reason="Requires Linux or Mac.",
+)
+@pytest.mark.parametrize("remote", [True, False])
+def test_driver_as_reader(ray_start_cluster, remote):
```

Comment:
Can you also add a test for driver reader + resizing? Seems like it should work but just to be safe.